--- pungi-3.03/pungi/gather.py.orig	2013-12-07 23:25:33.790299986 +0100
+++ pungi-3.03/pungi/gather.py	2013-12-07 23:27:06.347452833 +0100
@@ -1528,7 +1528,8 @@
 
         if self.tree_arch == 'i386' or self.tree_arch == 'x86_64':
             extraargs.extend(x86bootargs)
-            if self.tree_arch == 'x86_64':
+            if self.tree_arch == 'x86_64' and \
+                    os.path.exists(os.path.join(self.topdir, 'images', 'efiboot.img')):
                 extraargs.extend(efibootargs)
                 isohybrid.append('-u')
                 if os.path.exists(os.path.join(self.topdir, 'images', 'macboot.img')):
