--- pungi-3.12/pungi/__init__.py.orig	2015-03-23 13:24:45.173223580 +0100
+++ pungi-3.12/pungi/__init__.py	2015-03-23 13:24:59.222835656 +0100
@@ -1411,7 +1411,7 @@
 
         lorax.run(self.ayum, product=product, version=version, release=release,
                   variant=variant, bugurl=bugurl, isfinal=isfinal, domacboot=domacboot,
-                  workdir=workdir, outputdir=outputdir, volid=volid, installpkgs=installpkgs)
+                  workdir=workdir, outputdir=outputdir, volid=volid, installpkgs=installpkgs, doupgrade=False)
 
         # write out the tree data for snake
         self.writeinfo('tree: %s' % self.mkrelative(self.topdir))
