diff -ur revisor-2.2/revisor/cfg.py revisor-2.2-qubes/revisor/cfg.py
--- revisor-2.2/revisor/cfg.py	2010-06-04 15:36:03.000000000 +0200
+++ revisor-2.2-qubes/revisor/cfg.py	2011-04-04 00:19:59.098202616 +0200
@@ -1063,7 +1063,7 @@
         self.pkgorder_style = ""
 
         # Use fedora-release by default
-        self.release_pkgs = "^fedora-release.*$"
+        self.release_pkgs = "^qubes-release.*$"
         self.release_files = "eula.txt fedora.css GPL README-BURNING-ISOS-en_US.txt RELEASE-NOTES-en_US.html ^RPM-GPG img images stylesheet-images"
 
         # Installation Media Options
diff -ur revisor-2.2/revisor/splittree.py revisor-2.2-qubes/revisor/splittree.py
--- revisor-2.2/revisor/splittree.py	2010-06-04 15:33:36.000000000 +0200
+++ revisor-2.2-qubes/revisor/splittree.py	2011-04-04 00:20:36.876202697 +0200
@@ -85,7 +85,7 @@
         self.src_list = []
         self.shared_list = []
         self.reverse_sort_srpms=None
-        self.common_files = ['beta_eula.txt', 'EULA', 'README', 'GPL', 'RPM-GPG-KEY', 'RPM-GPG-KEY-beta', 'RPM-GPG-KEY-fedora']
+        self.common_files = ['beta_eula.txt', 'EULA', 'README', 'GPL', 'RPM-GPG-KEY', 'RPM-GPG-KEY-beta', 'RPM-GPG-KEY-fedora', 'RPM-GPG-KEY-qubes']
         self.logfile = []
 
     def getSize(self, path, blocksize=None):
