diff -ur lorax-22.1/src/pylorax/treebuilder.py lorax-22.1-patch/src/pylorax/treebuilder.py
--- lorax-22.1/src/pylorax/treebuilder.py	2014-12-13 11:20:56.000000000 +1300
+++ lorax-22.1-patch/src/pylorax/treebuilder.py	2016-03-17 15:14:40.539871906 +1300
@@ -227,7 +227,7 @@
             if backup:
                 initrd = joinpaths(self.vars.inroot, outfile)
                 os.rename(initrd, initrd + backup)
-            cmd = dracut + [outfile, kernel.version]
+            cmd = dracut + ["--kver", kernel.version, outfile]
             runcmd(cmd, root=self.vars.inroot)
 
             # ppc64 cannot boot images > 32MiB, check size and warn
