diff -ur lorax-22.1/share/runtime-cleanup.tmpl lorax-22.1-patch/share/runtime-cleanup.tmpl
--- lorax-22.1/share/runtime-cleanup.tmpl	2014-12-13 11:20:56.000000000 +1300
+++ lorax-22.1-patch/share/runtime-cleanup.tmpl	2016-03-27 04:21:37.534000000 +1300
@@ -27,7 +27,8 @@
 ## we don't run SELinux (not in enforcing, anyway)
 removepkg checkpolicy selinux-policy libselinux-utils
 ## anaconda has its own repo files
-removefrom fedora-release --allbut /etc/os-release
+removefrom fedora-release --allbut /etc/os-release /usr/lib/os-release \
+                                   /usr/lib/os.release.d/*
 removepkg fedora-release-rawhide
 ## no user accounts = no account management
 removepkg usermode usermode-gtk passwd shadow-utils
